tools/vp20compiler/Makefile

```diff
@@ -8,7 +8,7 @@ SRCS = nvvertparse.c \
 
 OBJS = $(SRCS:.c=.o)
 
-CFLAGS = -std=c99
+CFLAGS = -std=gnu99
```

Comment:
I included the 64-bit version, but I just had a thought: Perhaps I should replace it with the 32-bit version for maximum compatibility. What do you think?